### PR TITLE
post: cli 6.2

### PR DESCRIPTION
**Pull request by cexbrayat**
WIP

- [x] update post date
- [x] linter
- [x] Ivy support

### CODE REVIEW

File: _posts/2018-09-17-angular-cli-6.2.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: What's new in Angular CLI 6.2?
+author: cexbrayat
+tags: ["Angular 6", "Angular 5", "Angular", "Angular 2", "Angular 4", "Angular CLI"]
+description: "Angular CLI 6.2 is out! Read about the watch mode for libraries, Ivy support, the new linter options and more!!"
```
[the remark this answers is not in the document]
Comment (cexbrayat) [reply _posts/2018-09-17-angular-cli-6.2.md]: On a pas le droit d'avoir de l'enthousiasme c'est ça?

**Comment by cexbrayat** [comment]
CLI 6.2.0 is out, so I'll probably publish tomorrow or maybe monday morning.
But I just realized that I wrote about some features that will only be in cli 7.0... I'll remove some parts (terser and breaking change)...

File: _posts/2018-09-07-angular-cli-6.2.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+You can also now simply run:
+
+    ng lint src/app/app.component.ts
+
+if you want to lint just a file.
+
+To conclude this part about the linter,
+an option that disappeared in CLI 6.0 but existed before is also back: `--lint-fix`.
+This option can be used with every schematic,
+and will automatically fix the lint issues in the new files generated.
```
Comment (jnizet) [inline _posts/2018-09-07-angular-cli-6.2.md]: new generated files

File: _posts/2018-09-07-angular-cli-6.2.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+As you hopefully know if you read [our article](/2018/05/04/angular-cli-6.0/) about the release of Angular CLI 6.0.0,
+it is now possible to have multiple applications and libraries in your CLI project.
+But, as I noted in the article, a slightly annoying thing was that,
+when you made a change to the library source,
+you had to rebuild it manually if you wanted the rest of the project to see it,
+because there was no watch mode for `ng build` in a library.
+
+That's now no longer the case,
+and you can use `ng build --watch` for a library too,
+so the rest of your project will see the modifications without any manual steps anymore!
```
Comment (jnizet) [inline _posts/2018-09-07-angular-cli-6.2.md]: Does that mean that using ng server or ng build --watch on the app will also build the lib in watch mode, or rather that, if you want ng serve to also detect changes on the app, you need to use ng build --atch on the lib, and, in another terminal, ng serve for the app?

Comment (cexbrayat) [reply _posts/2018-09-07-angular-cli-6.2.md]: IIRC, you still have to run `ng build --watch` for the lib and `ng serve` for the app.

**Pull request merged**